lib/src/main/java/com/faskn/lib/ClickablePieChart.kt

```diff
@@ -62,6 +62,8 @@ class ClickablePieChart @JvmOverloads constructor(
     private var popupText: String? = null
     private var showPercentage = false
 
+    private var legendAdapter : LegendAdapter = LegendAdapter()
```

Comment:
Could you assign this to null and initialize somewhere else to avoid any possible memory leaks?